tests/click_tests/reset.py

```diff
@@ -0,0 +1,73 @@
+import shamir_mnemonic as shamir
+
+from trezorlib import messages
+
+from .. import buttons
+
+
+def confirm_wait(debug, startswith):
+    layout = debug.wait_layout()
+    assert layout.text.startswith(startswith)
+    debug.click(buttons.OK, wait=True)
+
+
+def confirm_read(debug, startswith):
+    layout = debug.read_layout()
+    assert layout.text.startswith(startswith)
+    debug.click(buttons.OK, wait=True)
+
+
+def set_selection(debug, button, diff):
+    layout = debug.read_layout()
+    assert layout.text.startswith("Slip39NumInput")
+    for _ in range(diff):
+        debug.click(button, wait=False)
+    debug.click(buttons.OK, wait=True)
+
+
+def read_words(debug, is_advanced=False):
+    # read words
+    words = []
+    layout = debug.read_layout()
+    if is_advanced:
+        assert layout.text.startswith("Group")
+    else:
+        assert layout.text.startswith("Recovery share")
+    for i in range(6):
+        debug.read_layout()
+        lines = debug.state().layout_lines
```

Comment:
Is there any reason not to call debug.read_layout?

Reply:
you are right, we can use debug.read_layout().lines to get the same result, fixed